src/domain/swap/errors.ts

```diff
@@ -1,6 +1,9 @@
 import { DomainError, ErrorLevel } from "@domain/shared"
 
 export class SwapError extends DomainError {}
+export class SwapErrorNonCritical extends DomainError {
+  level = ErrorLevel.Info
```

Comment:
this is the default afaik. no need to set it explicitly

Reply:
skimming through it, I think your issue is that you're throwing on `cron.ts` if an error is being returned. this is what is also trigger pager duty

Reply:
yep, i was following the other cron jobs as a template. should I not throw?